### PR TITLE
lxd: add an --image argument to cleanbuild

### PR DESCRIPTION
**Pull request by mwhudson**
A replacement for #1557

### CODE REVIEW

File: snapcraft/cli/env.py [hunk 1]
```diff
@@ -50,6 +50,11 @@ def use_container(self):
     def remote(self):
         return self._remote
 
+    @property
+    def image(self):
+        # TODO: allow this to be overridden
```
Comment (kalikiana) [inline snapcraft/cli/env.py]: It slightly bugs me that now there's two places where the default image is defined... how about putting the default image in snapcraft/internal/common.py?

**Comment by kalikiana** [comment]
I proposed  #1787 to fix the refresh test.

File: snapcraft/cli/env.py (hunk 1 above)
Comment (mwhudson) [reply snapcraft/cli/env.py]: Done

**Comment by mwhudson** [comment]
I updated the refresh command now that the test for it fails :)

**Comment by mwhudson** [comment]
Any update? I opened #1557 nearly four months ago now :-)

**Comment by sergiusens** [comment]
Moving forward, we have decided to move away from containers as the mechanism to execute builds https://forum.snapcraft.io/t/draft-consistent-build-environments-build-vms/5374/3

Sorry for the late reply and the wasted effort.

**Pull request closed without merging**